pkgs/development/libraries/fontconfig/default.nix

```diff
@@ -69,7 +70,7 @@ stdenv.mkDerivation rec {
     libxslt
     pkg-config
     autoreconfHook
-  ];
+  ] ++ stdenv.lib.optional stdenv.isDarwin CoreFoundation;
```

Comment:
Did you mean to add it to `buildInputs`?

Reply:
Yes, thanks. Still on my first coffee today ;)